src/app/registration/registration.component.spec.ts

```diff
@@ -1,14 +1,14 @@
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
-import { 
+import {
```

Comment:
Files like this, and a couple others in this PR, which do not change any functionality, but only do minor whitespace-type changes, should be in the own PR. In this case, it is fine, just something to keep in mind. Once PRs get big, it can easy to overlook stuff when unrelated changes like this are in the mix.

Reply:
Ok will do in the future!